metron-platform/metron-common/src/main/java/org/apache/metron/common/query/PredicateProcessor.java

```diff
@@ -18,12 +18,20 @@
 
 package org.apache.metron.common.query;
 
-import org.antlr.v4.runtime.*;
-import org.apache.metron.common.query.generated.*;
+import org.antlr.v4.runtime.ANTLRInputStream;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.antlr.v4.runtime.TokenStream;
+import org.apache.metron.common.query.generated.PredicateLexer;
+import org.apache.metron.common.query.generated.PredicateParser;
+
+import static org.apache.commons.lang3.StringUtils.isEmpty;
 
 
 public class PredicateProcessor {
   public boolean parse(String rule, VariableResolver resolver) {
+    if (isEmpty(rule)) {
```

Comment:
It appears that Hadoop Configuration resolves empty values to null - this handled both circumstances. I can add this for the PredicateProcessor and then add an additional translation in the client where the val is pulled from the Hadoop conf.